app/models/limber/tag_layout_template/walk_quadrants.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-module Limber::TagLayoutTemplate::WalkQuadrants # rubocop:todo Style/Documentation
+module Limber::TagLayoutTemplate::WalkQuadrants
   include Limber::TagLayoutTemplate::TagWalker
```

Comment:
And some automatic fixes. Not 100% sure why its decided this doesn't need documentation any more.